epigraphhub/models/ngboost_models.py

```diff
@@ -0,0 +1,740 @@
+#!/usr/bin/env python3
+"""
+Created on Fri Jan 28 14:55:40 2022
```

Comment:
maybe it would be nice to have more description about this module inside this docstring